src/modules/item/sagas.ts

```diff
@@ -454,6 +463,7 @@ export function* itemSaga(legacyBuilder: LegacyBuilderAPI, builder: BuilderClien
     }
     yield put(saveItemRequest(newItem, {}))
     yield take(SAVE_ITEM_SUCCESS)
+    yield put(closeModal('MoveItemToCollectionModal'))
```

Comment:
Question: is this `MoveItemToCollectionModal` related to this feature? Why do we need to close it? 🤔

Reply:
It's not related to this feature, but when removing the action: `SAVE_ITEM_SUCCESS` from the [modalSaga]((https://github.com/decentraland/builder/pull/2206/files#diff-36e062192786423af2f7cc0356f14a15104beff8969e0d53957d623e277f795eL5)), the modals that dispatch a `saveItemRequest` action are not closing automatically anymore